_posts/2023-02-15-adbc-0.2.0-release.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "Apache Arrow ADBC 0.2.0 (Libraries) Release"
+date: "2023-02-15 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 0.2.0 release of
+the Apache Arrow ADBC libraries. This covers includes [**34
+resolved issues**][1] from [**7 distinct contributors**][2].
+
+This is a release of the **libraries**, which are at version
+0.2.0.  The **API specification** is versioned separately and is
+at version 1.0.0.
+
+The release notes below are not exhaustive and only expose selected
+highlights of the release. Many other bugfixes and improvements have
+been made: we refer you to the [complete changelog][3].
+
+## Release Highlights
+
+An ADBC Flight SQL driver in Go has been added.  This driver has
+bindings for C/C++ and Python as well.  This lets us distribute and
+update the driver independently of PyArrow, without causing conflicts
+in dependencies like Protobuf and gRPC.
+
+The Go database/sql interface now returns standard library `time.Time`
+values for Arrow time and date columns.
+
+The PostgreSQL driver has support for more types.
+
+## Contributors
+
+```
+$ git shortlog --perl-regexp --author='^((?!dependabot\[bot\]).*)$' -sn apache-arrow-adbc-0.1.0..apache-arrow-adbc-0.2.0
+    56	David Li
+     8	Sutou Kouhei
+     7	Matt Topol
+     4	Jacob Marble
+     2	Benson Muite
+     1	Dave Hirschfeld
+     1	Jianfeng Mao
+```
+
+## Roadmap
+
+New extensions to the API specification have been proposed.  These
+will be backwards-compatible and will become API specification 1.1.0.
+For details, see the [mailing list discussion][adbc-api-1.1.0].
```

Comment:
should we mention the potential additional canonical options that have been waiting?

Reply:
Linked to the GitHub milestone as well.